homeassistant/components/sensibo/translations/en.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Translations are not part of our codebase anymore. Please rebase this PR onto the latest dev and remove the `en.json`.

../Frenck